.github/new-prs-labeler.yml

```diff
@@ -621,6 +618,10 @@ llvm:adt:
 llvm:support:
   - llvm/**/Support/**
 
+# Skip llvm/test/MC, which includes target-specific directories
+llvm:mc:
+  - llvm/lib/MC/**
```

Comment:
I was going to change this recently too, sorry I didn't get around to it before you did. I agree with your other proposed changes to the labels for PRs/Issues.

Please add:
- `llvm/include/llvm/MC/**`
- `llvm/tools/llvm-mc/**`

I agree on skipping `llvm/unittests/MC` for the same reasons as excluding the tests.

Reply:
Thanks for the suggestion! Done